python/pyspark/sql/connect/dataframe.py

```diff
@@ -480,9 +480,10 @@ def to_jcols(
 
     def hint(self, name: str, *params: Any) -> "DataFrame":
         for param in params:
-            if param is not None and not isinstance(param, (int, str)):
+            if param is not None and not isinstance(param, (int, str, float, list)):
```

Comment:
Let's add `float` only for now then.

Reply:
Awesome, will create a JIRA to add list later